dspace-api/src/main/resources/Messages.properties

```diff
@@ -901,17 +901,17 @@ jsp.search.filter.original_bundle_filenames                      = Filename
 jsp.search.filter.original_bundle_descriptions                   = File description
 jsp.sherpa.title = SHERPA/RoMEO Publisher Policy Database
 jsp.sherpa.loading = <p>Fetching policy information from the SHERPA/RoMEO database</p><img alt="loading" src="{0}/sherpa/image/ajax-loader-big.gif" />
-jsp.sherpa.heading = <p class="sherpaDisclaimer"><a href="http://www.sherpa.ac.uk/romeo.php" target="_blank"><img align="left" src="{0}/sherpa/image/romeosmall.gif" width="100" height="54" alt="SHERPA/RoMEO Database" border="0"></a> All SHERPA/RoMEO information is correct to the best of our knowledge but should not be relied upon for legal advice. SHERPA cannot be held responsible for the re-use of RoMEO data, or for alternative interpretations which are derived from this information.</p>
-jsp.sherpa.error = <p class="sherpaError">Sorry, we have had trouble querying the SHERPA/RoMEO Database. No data are availables, try later or check directly the <a href="http://www.sherpa.ac.uk/romeo.php" target="_blank">SHERPA/RoMEO WebSite</a>.</p>
-jsp.sherpa.noresult = <p class="sherpaNoResult">Sorry, there are not data in the <a href="http://www.sherpa.ac.uk/romeo.php" target="_blank">SHERPA/RoMEO Database</a> for the ISSNs that you have entered.</p>
-jsp.sherpa.oneresult = <p>The <a href="http://www.sherpa.ac.uk/romeo.php" target="_blank">SHERPA/RoMEO</a> Database provide the following data for the journal that you have entered.</p>
+jsp.sherpa.heading = <p class="sherpaDisclaimer"><a href="https://v2.sherpa.ac.uk/romeo/" target="_blank"><img align="left" src="{0}/sherpa/image/romeosmall.gif" width="100" height="54" alt="SHERPA/RoMEO Database" border="0"></a> All SHERPA/RoMEO information is correct to the best of our knowledge but should not be relied upon for legal advice. SHERPA cannot be held responsible for the re-use of RoMEO data, or for alternative interpretations which are derived from this information.</p>
+jsp.sherpa.error = <p class="sherpaError">Sorry, we have had trouble querying the SHERPA/RoMEO Database. No data is available, try later or check directly at the <a href="https://v2.sherpa.ac.uk/romeo/" target="_blank">SHERPA/RoMEO WebSite</a>.</p>
+jsp.sherpa.noresult = <p class="sherpaNoResult">Sorry, there is no data in the <a href="http://www.sherpa.ac.uk/romeo.php" target="_blank">SHERPA/RoMEO Database</a> for the ISSNs that you have entered.</p>
+jsp.sherpa.oneresult = <p>The <a href="https://v2.sherpa.ac.uk/romeo/" target="_blank">SHERPA/RoMEO</a> database provides the following data for the journal that you have entered.</p>
 jsp.sherpa.moreresults = <p>The ISSNs that you have entered match with multiple journals, please review them. You can find the publisher policy for these journals below.</p>
```

Comment:
I have changed the JSPUI handling so that each ISSN is now search separately, as with XMLUI. Additional results will be displayed below (since we are scrolling overflow, this is generally OK), as well as adding messages to indicate when there are multiple results, and/or when at least one ISSN returned an error or invalid response